app/views/merchants/invoices/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<center><h1>Little Esty Shop</h1></center><br>
+
+<h3><%= @merchant.name %></h3>
+
+<p>My Invoices</p>
+
+<% @merchant.unique_invoices.each do |invoice| %>
+  <p>Invoice <%= link_to "##{invoice.id}", "/merchants/#{@merchant.id}/invoices/#{invoice.id}" %></p>
```

Comment:
Could consider using a path helper for the link address

Reply:
Added refactor to this project card - will do after merge